lib/protocol/routes.js

```diff
@@ -700,6 +700,12 @@ const METHOD_MAP = {
       },
     }
   },
+
+  // chromium devtools
+  // https://chromium.googlesource.com/chromium/src/+/master/chrome/test/chromedriver/server/http_handler.cc
+  '/session/:sessionId/goog/cdp/execute': {
```

Comment:
`goog` is a vendor prefix in such case. It makes sense to replace it with a pattern matcher, like `:vendor`

Reply:
Good 👍 like it.